packages/sdk/src/middleware/middleware.ts

```diff
@@ -37,7 +38,16 @@ const actionToUri = (action: TSdkAction, projectKey?: string): string => {
   const service = requestBuilder[action.payload.service];
   if (action.payload.options) service.parse(action.payload.options);
 
-  return service.build();
+  return service.build({
+    // given `service=orderEdits` and given `applyOrderEditTo`, we build an apply endpoint
+    // given `service=orderEdits` and no `applyOrderEditTo`, we build an update endpoint
+    // https://docs.commercetools.com/api/projects/order-edits
+    applyOrderEdit:
+      action.payload.service === SUPPORTED_SERVICES.orderEdits &&
+      typeof action.payload.options?.applyOrderEditTo === 'string',
```

Comment:
we want to ensure that it is a string, not `null` nor a number

Reply:
There is one thing that I'm not sure I follow 100%. Is this parameter then going to be send for all the services and not just for `order-edits` (to `false` obviously) or is this part of the code only related to the interaction with the `order-edits` endpoint?

Reply:
This will always be sent to `service.build()`
https://github.com/commercetools/nodejs/blob/master/packages/api-request-builder/src/create-service.js#L128

Which by default is `false`.
We continue sending false as long as it doesn't fulfil the requirements discussed in the PR description